pkg/util/useragent_test.go

```diff
@@ -187,11 +178,12 @@ func TestParseUserAgents(t *testing.T) {
 				IsGRPC:              true,
 				HostName:            hostname,
 				UseGRPCStatusUpdate: true,
+				RuntimeID:           "",
```

Comment:
same as above, no reason to have a test with an empty runtimeID, but parsing tests without the section make sense